vminitd/Sources/vmexec/Mount.swift

```diff
@@ -44,6 +48,29 @@ struct ContainerMount {
         guard symlink("pts/ptmx", ptmx) == 0 else {
             throw App.Errno(stage: "symlink(pts/ptmx)")
         }
+
+        if process.terminal {
+            var buf = [CChar](repeating: 0, count: 4096)
+            let len = readlink("/proc/self/fd/0", &buf, buf.count - 1)
```

Comment:
Because we create the pty inside of the namespaces now, we should just be able to ptsname the fd

Reply:
We should keep it. Here's what I'd propose: leave `configureConsole` as is, and then we can have a `mountConsole` (or whatever you think is a good name) method that takes in the path to the slave side pty and then just bind mounts it over a /dev/console file we create. 
e.g:
```
mount(slavePath, "/dev/console", "", "bind", MS_BIND, "")
```

And the mount should be done right after we're done setting things up (we set ctty), but prior to sending the fd over the pipe to vminitd.